### PR TITLE
NEXUS-1034: Velocity is shared across nexus

### PR DESCRIPTION
**Pull request by cstamas**
This change bumps multiple Spice/Sisu components
(mailer, restlet-bridge), dumps some ancient ones (plexus-velocity)
and introduces the shiny new sisu-velocity.

While the bumped deps are all about "update" in majority (to use
new sisu-velocity), plexus-restlet-bridge holds the needed fix too,
that makes use share _one single_ instance of Velocity, instead
recreating it by each request.

### CODE REVIEW

**Comment by cstamas** [comment]
CI:
https://builds.sonatype.org/view/nexus/job/nexus-oss-its-feature/349/
(or around that, as job was executng)

File: nexus/pom.xml [hunk 1]
```diff
@@ -175,12 +175,12 @@
     <plexus-jetty6.version>1.7</plexus-jetty6.version>
     <plexus-jetty7.version>1.2.1</plexus-jetty7.version>
     <sisu-jetty8.version>1.2-SNAPSHOT</sisu-jetty8.version>
+    <sisu-velocity.version>1.0-SNAPSHOT</sisu-velocity.version>
     <plexus-jetty-testsuite.version>2.1</plexus-jetty-testsuite.version>
-    <plexus.restlet.bridge.version>1.20</plexus.restlet.bridge.version>
+    <plexus.restlet.bridge.version>1.3-SNAPSHOT</plexus.restlet.bridge.version>
```
Comment (nabcos) [inline nexus/pom.xml]: from 1.20 to 1.3?

Comment (cstamas) [inline nexus/pom.xml]: ouch!

**Comment by cstamas** [comment]
plexus-restlet-bridge mixup fixed, 1.21-snap is the real one

**Comment by cstamas** [comment]
CI voted +1
https://builds.sonatype.org/view/nexus/job/nexus-oss-its-feature/349/
(with some win non relevant failures, some test timing and FS+trash related stuff)

**Comment by adreghiciu** [comment]
+1

**Pull request merged**